vanhouse/src/App.js

```diff
@@ -1,36 +1,163 @@
-import React from 'react'
+import React,{useState}from 'react'
 import {BrowserRouter as Router, Route} from "react-router-dom";
-import {Container,Row,Col} from 'react-bootstrap';
+import {InputGroup, Button, FormControl, DropdownButton,ButtonToolbar,Container,Col,Row} from 'react-bootstrap';
+
+import {MenuItem,Dropdown} from 'bootstrap';
 import PostDetail from "./components/PostDetail";
 import Header from "./components/Header";
 import UserList from './components/UserList';
+import Price from './components/Price';
+import Location from './components/Location';
+import Category from './components/Category';
+import SearchBar from './components/SearchBar';
 import "./App.css";
 import PostCollection from "./components/PostCollection";
+import search from './assets/search.png';
 
 function App() {
+  const [leftState,setLeftState] = useState(1);
+  const [left,setLeft] = useState(<UserList />);
+  const [ul,setu] = useState(Number(0));
+  
+  
+
+
+
+
+
+
+
+  function handleClick(i){
+    
+    console.log("asdasd");
+    console.log(i);
 
-    return (
+    
+    setu(i);
+  }
+  
+  
+  
+  const handleReset = (i)=>{
+    
+  }
+
+  
```

Comment:
Here somehow they are not formatted. But in `project_3` branch now, they are formatted.